### PR TITLE
Update text.py with metadata

### PR DESCRIPTION
**Pull request by chinmaychahar**
- [x] issues: fixes #11765 

Added metadata to text.py in examples/plotting/text.py

### CODE REVIEW

**Comment by bryevdv** [comment]
@chinmaychahar It looks like maybe this PR branch was branched off the other PR branch rather than off `branch-3.2` can you rebase the changes on `branch-3.2` (you may need to force-push afterwards). Let me know if I can help with more details on those steps.

**Comment by chinmaychahar** [comment]
@bryevdv I realized there were some previous commits made to the `branch-3.2` itself in my fork so I'm not sure if rebasing would help. However, all these changes and corresponding PRs have been successfully merged. Should I consider reverting them from the current branch `metadata`? I think that should work.

File: examples/plotting/text.py [hunk 1]
```diff
@@ -1,3 +1,12 @@
+'''This example shows multiple variations in angle and alignment to multi-line text using CustomJS.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.model.CustomJS, bokeh.model.Slider
+    :refs: :ref:`bokeh.models.Slider.js_on_change`
+    :keywords: CustomJS, Slider, Text Angle, Text Align
```
Comment (bryevdv) [inline examples/plotting/text.py]:
```suggestion
    :keywords: customjs, slider, text angle, text align
```

File: examples/plotting/text.py [hunk 2]
```diff
@@ -1,3 +1,12 @@
+'''This example shows multiple variations in angle and alignment to multi-line text using CustomJS.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.model.CustomJS, bokeh.model.Slider
+    :refs: :ref:`bokeh.models.Slider.js_on_change`
```
Comment (bryevdv) [inline examples/plotting/text.py]: We try to link to the user's guide where possible
```suggestion
    :refs: :ref:`ug_interaction_js_callbacks_customjs`
```

**Review by bryevdv — COMMENTED** [review]
Thanks @chinmaychahar a few comments

**Comment by chinmaychahar** [comment]
Thanks for the comments, updated accordingly

**Review by bryevdv — APPROVED** [review]

**Comment by bryevdv** [comment]
Thanks @chinmaychahar !

**Pull request merged**